graph2tac/loader/predict_server.py

```diff
@@ -319,15 +342,18 @@ def check_alignment(msg):
             unalignedDefinitions = unaligned_definitions,
         )
 
-def prediction_loop(predict_server, capnp_socket, record_file):
-    if predict_server.config.with_meter:
-        capnp_socket = tqdm.tqdm(capnp_socket)
+def prediction_loop(predict_server: PredictServer, capnp_socket: socket.socket, record_file: Optional[IO]):
```

Comment:
Shouldn't `IO` be `BinaryIO`?

Reply:
Good point.  I was unsure how strong of a contract `capnp_message_generator` has on record_file, but I should have just looked at the code.  Fixed.

Reply:
As an aside: Is your editor able to retrieve the type annotations and documentation from `data_reader.pyx`? Mine isn't, which is extremely annoying...

Reply:
No.  But I think you can add type stubs to make this information available.  I was going to approach you about it.